very easy/10.js

```diff
@@ -1,5 +1,9 @@
 function divisibleByFive(n) {
-
+    if(n%5 == 0){

```

Comment:
Zamiast 
`if(n%5 == 0){
        return true;
    }else{
        return false;
    }`

Możesz skrócić to do zapisu:
`return n%5 == 0`